vars/mavenCI.groovy

```diff
@@ -61,8 +61,16 @@ def call(body) {
             }
         }
 
+        def testCmd = config.integrationTestCmd
+        if(config.runTestsInUserNamespace == true && testCmd) {
+            def nameSpace = utils.getUsersNamespace()
+            echo "Running the integration tests in the namespace: ${nameSpace}"
```

Comment:
nameSpace -> namespace
and we don't need "the" in  "... in the namespace: ${namespace}"